vms/authentication/tests/test_services.py

```diff
@@ -1,3 +1,14 @@
 from django.test import TestCase
+from authentication.views import login_process
+from django.test.client import RequestFactory
+
 
 # Create your tests here.
+class TestLoginProcess(TestCase):
+
+	def testLoginViewRateLimit(self):
```

Comment:
@smarshy @tapasweni-pathak 
I updated this PR, but tests are still failing :sob: